packages/lesswrong/lib/voting/votingSystems.tsx

```diff
@@ -149,134 +149,6 @@ export function getVoteAxisStrength(vote: DbVote, usersById: Record<string,DbUse
   return calculateVotePower(user.karma, voteType);
 }
 
-export type ReactBallotAxis = {
-  name: string,
-  scoreLabel: string,
-  goodLabel: string,
-  badLabel: string,
-}
-export type ReactBallotStandaloneReaction = {
-  name: string,
-  label: string,
-  icon: string,
-}
-export const reactBallotAxes: ReactBallotAxis[] = [
-  {name: "truth", scoreLabel: "Truth", goodLabel: "True", badLabel: "False"},
-  {name: "aim", scoreLabel: "Aim", goodLabel: "Hits the Mark", badLabel: "Misses the Point"},
-  {name: "clarity", scoreLabel: "Clarity", goodLabel: "Clear", badLabel: "Muddled"},
-  {name: "seeking", scoreLabel: "Seeking",goodLabel: "Seeks Truth", badLabel: "Seeks Conflict"},
-];
-export const reactBallotStandaloneReactions: ReactBallotStandaloneReaction[] = [
-  {name: "skepticism", label: "Skepticism", icon: "🤨"},
-  {name: "enthusiasm", label: "Enthusiasm", icon: "🎉"},
-  {name: "empathy",    label: "Empathy",    icon: "❤️"},
-  {name: "surprise",   label: "Surprise",   icon: "😮"},
-]
-
-const reactBallotAxisNames = reactBallotAxes.map(axis=>axis.name);
-const reactBallotStandaloneReactionNames = reactBallotStandaloneReactions.map(reaction => reaction.name);
-
-registerVotingSystem({
```

Comment:
@jimrandomh I believe we don't use this or emojiReactions anymore, right? Your name is on the blame so thought I'd check with you

Reply:
EA Forum still has posts using these voting systems

Reply:
Cool, I've put them back in